ES/ESmy016.xml

```diff
@@ -141,12 +142,12 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                                  </item>
 
                                  <item xml:id="q6" n="6">
+                                   <note> (the structure of the quire
+                                    cannot be ascertained;
+                                    leaf 1 (<locus target="#38"/>) is possibly single.)</note>
```

Comment:
I had to move the note up, otherwise the app did not recognize the syntax and did not produce the diagram

Reply:
ok

Reply:
I was thinking of this. It is a border case because the biggest part of the texts in part 2 is in Hand 2 and the colophon 2 is also in Hand 2 (unless I am wrong assuming this), and the commissioner is the same. The leaf 43 indeed looks like a flyleave. A slight difference in layout is not so decisive if the size of the leaves is the same (cp MY-019). Perhaps the manuscript was a project that dragged on in time, a little bit, and was completed largerly by the same person. Of course you can say that 30 miracles of Mary + John the Baptist collection is an unlikely constellation, and to disprove this I would need to make a research. Well, sure, if you want, you can rearrange it into a composite one.